submissions/blobert-puzzle

```diff
@@ -0,0 +1,27 @@
+# Blobert Puzzle
+
+### Project Summary
+Give a short summary of your game project.
+> Unlock the mysteries of the Blobert puzzle. In this game, each Blobert NFT owner possesses a piece of a collective puzzle. Collaborate to piece together the puzzle and reveal the hidden masterpiece.
+
+### GitHub
+Provide the GitHub where your project is hosted. Please ensure your github repo is registered with [OnlyDust](https://app.onlydust.com/p/create) to receive awards.
+> https://github.com/Await-0x/blobert-puzzle
+
+### Play
+Provide instructions on how to play your game. It could be a URL, a README, or a link to a binary to install.
+> Play on Starknet mainnet: https://blobertpuzzle.xyz/
+
+### Twitter
+Provide the projects twitter handle. Please share your submission on socials too for more exposure!
+> https://twitter.com/await_0x
+
+### Team members
+Provide a list of your team members.
+> Await - https://github.com/Await-0x
+
+### Submission Tracks
+Provide a list of the tracks your project is submitting to.
+> - Road to Mainnet 
+> - Blobert / Realms 
+> - New World
```

Comment:
```suggestion
```

This doesn't qualify for the New World track since it is inside the Realms lore.